roles/ocitysmap/tasks/main.yml

```diff
@@ -0,0 +1,266 @@
+
+- name: Date import layers osmfr
+  find:
+    paths: /srv/{{ application_user_account }}/styles-commons/osmfr-layers
+    patterns: 'z7.tif'
+    age: -1d
+  register: find_layers
+
+- name: téléchargement des données layers osmfr
+  get_url:
+    url: http://osm13.openstreetmap.fr/~cquest/z7.tif
+    dest: /srv/{{ application_user_account }}/styles-commons/osmfr-layers
+    force: yes
+    mode: 0644
+  when: find_layers.matched == 0
+
+- name: lien pour acces au fichier z7.tif
+  file: 
+    path: /srv/{{ application_user_account }}/styles/osmfr-cartocss/layers
+    src: /srv/{{ application_user_account }}/styles-commons/osmfr-layers
+    state: link 
+    force: yes
+- name: lien pour acces donnees de base carto
+  file: 
+    path: /srv/{{ application_user_account }}/styles/osmfr-cartocss/data 
+    src: /usr/share/openstreetmap-carto-common/data
+    state: link 
+    force: yes
+
+#2# psql -f additions.sql
+#3# psql -f functions.sql (bug PARALLEL SAFE)
+#psql -f support/setup-maposmatic-admin.sql
+- name: complements a la base postgresql
+  command: "{{ item }}"
+  with_items:
+    - /usr/bin/psql -f additions.sql
+    - /usr/bin/psql -f functions.sql
+  become_user: "{{ application_user_account }}"
+  args:
+    chdir: /srv/{{ application_user_account }}/styles/osmfr-cartocss
+
+#4a# dans osmfr.yml remplacer par gis et mdpa
+- name: edition de osmfr.yml id
+  lineinfile:
+    dest: /srv/{{ application_user_account }}/styles/osmfr-cartocss/osmfr.yml
+    regexp: "^\\s+user: fr"
+    line: "    user: {{ db_owner_account }}"
+    state: present
+
+- name: edition de osmfr.yml pwd
+  lineinfile:
+    dest: /srv/{{ application_user_account }}/styles/osmfr-cartocss/osmfr.yml
+    regexp: "^\\s+password: .+$"
+    line: "    password: {{ db_owner_password }}"
+    state: present
+
+- name: edition de osmfr.yml dbname
+  lineinfile:
+    dest: /srv/{{ application_user_account }}/styles/osmfr-cartocss/osmfr.yml
+    regexp: "^\\s+dbname: \".+\"$"
+    line: "    dbname: {{ db_name }}"
+    state: present
+
+#4b# python -c 'import sys, yaml, json; json.dump(yaml.safe_load(sys.stdin), sys.stdout, indent=4, separators=(",", ": "))' < osmfr.yml > project.json.mml
+- name: utilisation du fichier osmfr.yml
+  shell: "python -c \"import sys, yaml, json; json.dump(yaml.safe_load(sys.stdin), sys.stdout, indent=4, separators=(',', ': '))\" < osmfr.yml > project.json.mml"
+  args:
+    chdir: /srv/{{ application_user_account }}/styles/osmfr-cartocss
+
+#5# curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -  (installation de nodejs et npm depuis un depot particulier)
+#old: apt-get install nodejs 
+#old: sudo apt-get install -y npm
+#old: sudo ln -s /usr/bin/nodejs /usr/bin/node
+#npm install -g @mapbox/carto
+#npm install -g millstone
+- name: installation de nodejs, npm, carto et millstone
+  shell: "{{ item }}"
+  with_items:
+    - curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash
```

Comment:
node 6.x is quite old. Did you try with latest node version ?

Reply:
look at 
http://linuxbsdos.com/2017/06/26/how-to-install-node-js-lts-on-debian-9-stretch/

not better than node-js 6.x
and the page is timestamped 2017

Reply:
ok i found that
https://github.com/nodesource/distributions#deb

i'll update the script

Reply:
According to https://github.com/nodejs/Release, version 8 is the latest LTS release.